src/app/public/src/modules/nav/nav.component.html

```diff
@@ -2,7 +2,7 @@
   <ol class="stache-nav-list">
     <li class="stache-nav-list-item"
       *ngFor="let route of routes"
-      [ngClass]="{'active': isActive(route), 'current': isCurrent(route)}">
+      [ngClass]="{'active': route.isActive, 'current': route.isCurrent}">
```

Comment:
Is there much of a performance or some other gain we are getting from this?  Feels strange to have these boolean values on the link model to me.